docs/dev/code_review.md

```diff
@@ -0,0 +1,121 @@
+# Code Reviews 
+
+Reviewing code is like proof reading a paper, it essential to ensure the followig: 
+
+- code is user readable
+- follows consistency
+- helps create collaboration
+- structure is proper
+
+!!! Note
+    Code review is not meant for debugging or finding bugs
+
+Anyone can code review even if you don't program in the native language, which is python for pyDARN. 
+Doing code reviews can also be good learning experiences to better understand the software and how to program. 
+
+Code reviews should not take more than an hour and Pull Requests should be less than 400 lines.
+However, this is challenging for new features and major changes. 
+If this is the case, then code review parts or specific files to reduce your time spent. 
+
+## How to start a Code Review 
+
+1. Pick a Pull Request to code review, which can be found [here](https://github.com/SuperDARN/pydarn/pulls)
+2. Click on `files chaned` at the top of the PR conversation below the title on the right side 
+3. Now review the changes that occurred in this PR - avoid commenting on other lines as this is scope creep.
+4. If you want to make a comment hover the cursor over the line numbers on the left side and a **+** button will appear. Click it!
+5. Now right any feedback or questions in the box and hit `Start a review` on the bottom right in the pop up window. 
+    
+    !!! Warning
+        Do not make several single comments as this can cause a lot of email notifications
+
+6. If you want to change a line in the code use [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/)
+7. If you wish comment on several consecutive lines hold **+** down and drag the cursor down to select multiple lines
+8. Once you are done, click on `Finish your changes` or `Review channges` (if no comments made) then comment on any general feedback in the text box
+9. Select:  
+  - Comment: general comments to be fixed up but nothing major
+  - Approve: everything looks good no changes needed
+  - Request Changes: major changes to make
+      
+    !!! Warning
+        Request changes, present anyone from merging the code. Generally you will need to approve the changes that were addressed. 
+        Generally if you cannot do this then do not use Request changes.
+
+10. Submit! 
+
+!!! Note
+    If you want to make minor changes directly in the code like typos and grammar changes you can pull the code down described in [testing](testing.md)
+    and change the code as you want and then re-push the code back to the code. The Developer can review the changes in your commit. 
```

Comment:
```suggestion
    and change the code as you want and then re-push the code back to the branch. The Developer can review the changes in your commit. 
```
I'm not sure what this note is trying to say, is it saying how to make changes if you have to go into the files again?

Reply:
@Devin-Huyghebaert suggested it like you could have checked out this branch and made changes within the file?

Reply:
Ah okay, so code is good!